external/fv3fit/tests/test_models.py

```diff
@@ -107,6 +108,26 @@ def test_DenseModel_clipped_inputs():
     xr.testing.assert_allclose(prediction_nan_filled, prediction_clipped, rtol=1e-3)
 
 
+def test_loaded_DenseModel_predicts_with_clipped_inputs(tmpdir):
+    hyperparameters = DenseHyperparameters(
+        ["a", "b"],
+        ["c"],
+        packer_config=PackerConfig({"a": {"z": SliceConfig(None, 3)}}),
+    )
+    model = DenseModel(["a", "b"], ["c"], hyperparameters)
+
+    nz = 5
+    dims = ["x", "y", "z"]
+    shape = (2, 2, nz)
+    arr = np.arange(np.prod(shape)).reshape(shape).astype(float)
+    input_data = xr.Dataset({"a": (dims, arr), "b": (dims, arr), "c": (dims, arr + 1)})
+    model.fit([input_data])
+    output_path = str(tmpdir.join("trained_model"))
+    fv3fit.dump(model, output_path)
+    model_loaded = fv3fit.load(output_path)
+    model_loaded.predict(input_data)
```

Comment:
Please perform prediction before and after dump/load-ing, and assert that the answer is unchanged.